### DISCUSSION TITLE
修改 quickjs 以导入 rust 函数 —— 一种扩展北海的新思路

### DISCUSSION THREAD

**Discussion by gcxfd** [openkraken/kraken#1372] [Show and tell]
[修改 quickjs 以导入 rust 函数 —— 一种扩展北海的新思路](https://rmw.link/log/2022-04-29-quickjs-rust)

**Comment by andycall** [comment]
emm， 看上去很意思。。

**Reply by gcxfd** [reply]
@andycall 早日支持  MutationObserver  https://github.com/openkraken/kraken/issues/1346 吧，我想用svelte写点东西玩玩。
